dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoListTreeTableItem.java

```diff
@@ -156,7 +156,7 @@ private void buildChildren(final ObservableList<TreeItem<JobInfoModel>> children
                         }
                     } else {
                         final GetObjectsWithFullDetailsSpectraS3Request request = new GetObjectsWithFullDetailsSpectraS3Request().withBucketId(modelValue.getBucket()).withIncludePhysicalPlacement(true);
-                        request.withName(modelValue.getName() + StringConstants.PERCENT).withFolder(modelValue.getName());
+                        request.withName(modelValue.getName() + StringConstants.PERCENT);
```

Comment:
Everything still worked with this removed?

Reply:
Let me validate tomorrow morning, but I think I tested everything.

Reply:
Ok, I ran some more tests. This code isn't exactly clean but I can't think of a condition  that we haven't exercised in the job recovery code.